0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -115,6 +128,34 @@ Content-Type: application/spsp4+json
 }
 ```
 
+In case of a pull payment server, it is possible to request the payment token via
```

Comment:
Yes, this needs better wording.

```suggestion
In case of a pull payment server, it is possible to request information about a particular payment that can be pulled from this server using the corresponding token, i.e.
```